packages/build-ui/src/components/all_filter_field.rs

```diff
@@ -0,0 +1,24 @@
+use bdk::prelude::*;
+use by_components::icons::arrows;
+
+#[component]
+pub fn AllFilterField(#[props(default = false)] show: bool) -> Element {
+    if !show {
+        return rsx! {
+            div {}
+        };
+    }
+
+    rsx! {
+        div { class: "relative",
+            div { class: "absolute inset-y-0 right-0 flex items-center pr-3 pointer-events-none",
+                arrows::ChevronDown { class: "[&>path]:stroke-white", height: 20, width: 20 }
+            }
+            input {
+                class: "text-[15px]/[23px]  border border-neutral-80 outline-none text-white hover:border-primary focus:border-primary aria-invalid:border-pink placeholder-neutral-800 disabled:!border-neutral-80 pl-3 p-2.5",
```

Comment:
```suggestion
                class: "text-[15px]/23  border border-neutral-80 outline-none text-white hover:border-primary focus:border-primary aria-invalid:border-pink placeholder-neutral-800 disabled:!border-neutral-80 pl-3 p-2.5",
```
If spacing is set to 1px, you can just describe number instead of `[value]`.
https://tailwindcss.com/docs/line-height#removing-the-leading